File: src/sharp/Module.TerritoryData.Library/TerritoryDataRepository.cs
using Module.TerritoryData.Library.Interfaces;
using Module.TerritoryData.Library.Models;
using Module.TerritoryData.Library.Models.Hierarchy;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Module.TerritoryData.Library
{
    internal class TerritoryDataRepository : ITerritoryDataRepository
    {
        public Address GetAddress(string addressCode)
        {
            throw new NotImplementedException();
        }

        public AddressList GetAddressList(SearchParams searchParams)
        {
            throw new NotImplementedException();
        }

        public City GetCity(string cityCode)
        {
            throw new NotImplementedException();
        }

        public CityList GetCityList(SearchParams searchParams)
        {
            throw new NotImplementedException();
        }

        public Country GetCountry(string countryCode)
        {
            throw new NotImplementedException();
        }

        public CountryList GetCountryList()
        {
            CountryList countryList = new CountryList();
            countryList.Add(new Country() { Id = 1, Code = "pl", Name = "Poland" });
            countryList.Add(new Country() { Id = 1, Code = "de", Name = "Germany" });
            return countryList;
        }

        public Level1Division GetLevel1Division(string zone1Code)
        {
            throw new NotImplementedException();
        }

        public Level1DivisionList GetLevel1DivisionList(SearchParams searchParams)
        {
            throw new NotImplementedException();
        }

        public Level2Division GetLevel2Division(string zone2Code)
        {
            throw new NotImplementedException();
        }

        public Level2DivisionList GetLevel2DivisionList(SearchParams searchParams)
        {
            throw new NotImplementedException();
        }

        public Level3Division GetLevel3Division(string zone3Code)
        {
            throw new NotImplementedException();
        }

        public Level3DivisionList GetLevel3DivisionList(SearchParams searchParams)
        {
            throw new NotImplementedException();
        }
    }
}

File: src/java/module.territorydata/src/main/java/pl/digihome/module/territorydata/library/TerritoryDataRepository.java
package pl.digihome.module.territorydata.library;

import pl.digihome.module.territorydata.library.interfaces.*;
import pl.digihome.module.territorydata.library.models.*;

public class TerritoryDataRepository implements ITerritoryDataRepository
{
	public final Address Address(String addressCode)
	{
		throw new UnsupportedOperationException();
	}

	public final AddressList GetAddressList(SearchParams searchParams)
	{
		throw new UnsupportedOperationException();
	}

	public final City GetCity(String cityCode)
	{
		throw new UnsupportedOperationException();
	}

	public final CityList GetCityList(SearchParams searchParams)
	{
		throw new UnsupportedOperationException();
	}

	public final Country GetCountry(String countryCode)
	{
		throw new UnsupportedOperationException();
	}

	public final CountryList GetCountryList()
	{
		CountryList countryList = new CountryList();
		Country tempVar = new Country();
		tempVar.setId(1);
		tempVar.setCode("pl");
		tempVar.setName("Poland");
		countryList.add(tempVar);
		Country tempVar2 = new Country();
		tempVar2.setId(1);
		tempVar2.setCode("de");
		tempVar2.setName("Germany");
		countryList.add(tempVar2);
		return countryList;
	}

	public final Zone1 GetZone1(String zone1Code)
	{
		throw new UnsupportedOperationException();
	}

	public final Zone1List GetZone1List(SearchParams searchParams)
	{
		throw new UnsupportedOperationException();
	}

	public final Zone2 GetZone2(String zone2Code)
	{
		throw new UnsupportedOperationException();
	}

	public final Zone2List GetZone2List(SearchParams searchParams)
	{
		throw new UnsupportedOperationException();
	}

	public final Zone3 GetZone3(String zone3Code)
	{
		throw new UnsupportedOperationException();
	}

	public final Zone3List GetZone3List(SearchParams searchParams)
	{
		throw new UnsupportedOperationException();
	}
}